src/screens/SignUp/index.tsx

```diff
@@ -36,16 +44,41 @@ function SignUp() {
     console.log(data);
   };
 
+  const changeLanguage = (lang: string): void => {
+    i18next.changeLanguage(lang);
+    setLanguage(lang);
+  };
+
   return (
     <div className="row full-height center middle">
       <form
         className={`column full-height center space-around ${styles.formContainer}`}
         onSubmit={handleSubmit(onSubmit)}
       >
-        <img className={styles.formImage} src={wLogo} alt="wolox-logo" />
+        <div className="full-width row center">
+          <button
+            className={`${styles.languageButton} ${language === 'es' ? styles.active : ''} m-right-1`}
```

Comment:
For these conditional classes we suggest use library classnames [here](https://www.npmjs.com/package/classnames)